index.js

```diff
@@ -0,0 +1,98 @@
+let clickedCard = null;
+let preventClick = false;
+let combosFound = 0;
+let clickcount = 0;
+let finalTime = 0;
+
+//TIMER CODE
+let startTime = Date.now();
+let second = 1000;
+let minute = second * 60;
+let hour = minute * 60;
+let timerCount = document.getElementById('timer');
+
+function stopTimer() {
+    clearInterval(timer);
+}
+
+function pad(n){
+  return ('00' + n).slice(-2);
+}
+
+let timer = setInterval(function(){
+  let currentTime = Date.now();
+  let difference = currentTime - startTime;
+
+  let hours = pad((difference / hour) | 0);
+  let minutes = pad(((difference % hour) / minute) | 0);
+  let seconds = pad(((difference % minute) / second) | 0);
+
+  finalTime = timerCount.innerHTML = 'TIME: ' + hours + ':' + minutes + ':' + seconds;
+
+// This only represents time between renders. Actual time rendered is based
+// on the elapsed time calculated above.
+}, 250);
+
+//GAME CODE
+const pictures = [
+    'pic1',
+    'pic2',
+    'pic3',
+    'pic4',
+    'pic5',
+    'pic6'
+]
+
+//ramdomize pictures at the start of the game
+const cards = [...document.querySelectorAll('.card')];
+for (let picture of pictures) {
+const cardAIndex = parseInt(Math.random() * cards.length);
+    const cardA = cards[cardAIndex];
+    cards.splice(cardAIndex, 1);
+    cardA.className += ` ${picture}`;
+    cardA.setAttribute('data-color', picture);
+
+    const cardBIndex = parseInt(Math.random() * cards.length);
+    const cardB = cards[cardBIndex];
+    cards.splice(cardBIndex, 1);
+    cardB.className += ` ${picture}`;
+    cardB.setAttribute('data-color', picture);
+}
+
+function onCardClicked(e) {
+    const target = e.currentTarget;
+    clickcount++;
+    document.getElementById("p1").innerHTML = ("CLICKS: " + clickcount);
+
+    if (preventClick || target === clickedCard || target.className.includes('done')) {
```

Comment:
what are these conditions supposed to mean? If they are invalid clicks, then set them to a variable.

`isInvalidClick = preventClick || target === ...`

Reply:
Also, please note that even though the click is invalid, you're still counting the click in the clickCounter